comms.js

```diff
@@ -37,8 +37,8 @@ Comms.prototype.write = function(msg) {
 	}
 };
 
-Comms.prototype.listen = function(port) {
-	var self = this, server = this.server = jot.createServer(port);
+Comms.prototype.listen = function(port, opts) {
+	var self = this, server = this.server = jot.createServer(opts || {}, port);
```

Comment:
Should we assume that port is a real number (`123`) or should we handle strings as well (`'123'`)